privacyIDEAADFSProvider/AuthPage.html

```diff
@@ -52,6 +52,36 @@
                 document.forms["loginForm"].submit();
             }
 
+            // WEBAUTHN FUNCTIONS
+            function uint6ToB64 (nUint6) {
```

Comment:
What?

Do we really need to implement such stuff? Isn't there a lib function for such things?
Or is it some recurring javascript, which we can put and import somewhere?

Reply:
i just copied the contents of our webauthn.js to the AuthPage.html since im not sure if there is a "official" way to get our js loaded since we only return html as a string.
I have some ideas in mind which might solve this but are a bit hacky. But for the first version, i just copied the code.

Reply:
OK, so this is probably some long term task.